sql/ddl_triggers.sql

```diff
@@ -74,8 +79,34 @@ BEGIN
                 AND trigger_name != '_timescaledb_main_after_insert_trigger'
```

Comment:
could we have a `get_timescale_triggers()` function and then do `where trigger_name <> ALL(get_timescale_triggers())` ?